Core/src/org/sleuthkit/autopsy/modules/embeddedfileextractor/SevenZipExtractor.java

```diff
@@ -724,7 +724,7 @@ boolean unpack(AbstractFile archiveFile, ConcurrentHashMap<Long, Archive> depthM
                 if (checkForIngestCancellation(archiveFile)) {
                     return false;
                 }
-                final String uniqueExtractedName = FileUtil.escapeFileName(uniqueArchiveFileName + File.separator + (inArchiveItemIndex / 1000) + File.separator + inArchiveItemIndex + "_" + new File(pathInArchive).getName());
+                final String uniqueExtractedName = FileUtil.escapeFileName(uniqueArchiveFileName + File.separator + (inArchiveItemIndex / 1000) + File.separator + inArchiveItemIndex);
```

Comment:
@APriestman perhaps we can take this a bit further and truncate other parts of the path as well and make sure that it is as short as possible. Like uniqueArchiveFileName for example, that one can be quite long as well? In getUniqueName() we do file.getName() + "_" + file.getId(). Perhaps we can just do file.getId() instead?

Reply:
Seems like that should work - either way it should correspond to exactly one file in the case. I'll do some testing.

Reply:
Great. The problem in production is that the case names are typically VERY long, so that doesn't leave much space for path elements inside the case output folder. We should reduce the path length as much as possible.